org-brain.el

```diff
@@ -205,11 +205,16 @@ its own line. If nil (default), children are filled up to the
 (defun org-brain-id-exclude-taggedp (id)
   "Return t if ID is tagged as being excluded from org-brain."
   (org-with-point-at (org-id-find id t)
-    (let ((tags (org-get-tags-at)))
-      (or (member org-brain-exclude-tree-tag tags)
-          (and (member org-brain-exclude-children-tag tags)
-               (not (member org-brain-exclude-children-tag
-                            (org-get-tags-at nil t))))))))
+	(org-brain-entry-at-point-excludedp)))
+
+(defun org-brain-entry-at-point-excludedp ()
+  "Return t if the entry at point is tagged as being excluded
+from org-brain."
+  (let ((tags (org-get-tags-at)))
+	(or (member org-brain-exclude-tree-tag tags)
+		(and (member org-brain-exclude-children-tag tags)
+			 (not (member org-brain-exclude-children-tag
+						  (org-get-tags-at nil t)))))))
```

Comment:
I guess `org-get-tags` could be used instead of `(org-get-tags-at nil t)`. Don't know if it matters though.

Reply:
I haven't found a compelling reason to use one over the other. So, I'll leave it as-is.